### ISSUE TITLE
Crates empty

### ISSUE THREAD

**Issue by LudoFriend** [FlodGamedev/OceanCraftBugtracker#42]
All the crates lost their contents as did the fish bowl.

**Comment by LudoFriend** [comment]
They appear to have stabilised

**Comment by Th3T3chW1z** [comment]
Oculus quest 2. The storage crates lost all their items after leaving the game and then returning.

**Comment by ClodeFlodlab** [comment]
Hello,
we had an issue with the save system of build objects (crate, plants, fish, grill, etc.) we optimised this in the new Update 0.7x - Islands. Things should now stay as they are on your craft.

**Comment by LudoFriend** [comment]
Hi,
I’m still having this issue. My friend and I collected lots of items and put them in storage crates and we also planted vegetables. We came out of the game for a short while and when we went back, everything from the crates, benches floors and planters had gone.

Loving the new updates with the islands, Easter eggs etc.

Sent from my iPhone

On 12 Apr 2022, at 17:43, ClodeFlodlab ***@***.***> wrote:

﻿

Hello,
we had an issue with the save system of build objects (crate, plants, fish, grill, etc.) we optimised this in the new Update 0.7x - Islands. Things should now stay as they are on your craft.
